arangosh/Restore/RestoreFeature.cpp

```diff
@@ -1539,6 +1539,13 @@ void RestoreFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opt
                   new UInt32Parameter(&_options.threadCount))
       .setIntroducedIn(30400);
 
+  options
+      ->addOption("--initial-connect-retries",
+          "number of connect retries for initial connection",
+          new UInt32Parameter(&_options.initialConnectRetries))
+      .setIntroducedIn(30800)
```

Comment:
```suggestion
      .setIntroducedIn(30801)
```

Reply:
@Simran-B : what is our convention here? Should we actually list previous versions here too (i.e. mention `30713` **and** `30801`, or should we just mention `30801` in 3.8?)